packages/math-input/src/components/keypad/index.js

```diff
@@ -1,8 +1,5 @@
 //@flow
-import Clickable from "@khanacademy/wonder-blocks-clickable";
-import Color from "@khanacademy/wonder-blocks-color";
 import {View} from "@khanacademy/wonder-blocks-core";
-import {StyleSheet} from "aphrodite";
```

Comment:
All of this stuff just wasn't being used?  🙃

Reply:
Correct. It was all greyed out and was triggering our ESLint `no-unused` rule.